### ISSUE TITLE
Impossible to install ABT 5.2.2 with Wordpress version 5.2.4

### ISSUE THREAD

**Issue by natoinet** [dsifford/academic-bloggers-toolkit#584]
**ABT Version:** I wanted to install ABT v5.2.2 

**PHP Version:** 7.3 

**Theme:** Twenty Nineteen v1.4

**Expected behavior:** I wanted to install the plugin from Wordpress admin. However, the minimum Wordpress required version is 5.3, when the latest Wordpress release is 5.2.4.

**Comment by dsifford** [comment]
Yep, that's expected. 5.3 is slated to release in a couple weeks and at that time it should open up and allow you to install it when you update.

The plugin breaks the editor in earlier versions of WordPress due to using newer features that aren't available in those versions.

For more info, see #582 

Sit tight!
